api/base/pagination.py

```diff
@@ -172,6 +172,13 @@ class MaxSizePagination(JSONAPIPagination):
     max_page_size = None
     page_size_query_param = None
 
+
+class ElasticsearchListViewPagination(JSONAPIPagination):
```

Comment:
naming: this name sounds like what one should use for paginating any `ElasticsearchListView` response, but in fact is the very special/expensive case of an 10000-all-at-once download -- clearer naming, pls

Reply:
I disagree it has a very clear name.